### ISSUE TITLE
trouble on CentOS 6.7 with  python 2.6

### ISSUE THREAD

**Issue by arm2arm** [glamp/bashplotlib#29]
After installling I am getting following error:

```bash
 git clone https://github.com/glamp/bashplotlib.git
 cd bashplotlib
 python setup.py install


bashplotlib]$ hist --demo
Traceback (most recent call last):
  File "/usr/bin/hist", line 9, in <module>
    load_entry_point('bashplotlib==0.6.3', 'console_scripts', 'hist')()
  File "/usr/lib/python2.6/site-packages/pkg_resources.py", line 299, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.6/site-packages/pkg_resources.py", line 2229, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.6/site-packages/pkg_resources.py", line 1948, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
  File "/usr/lib/python2.6/site-packages/bashplotlib-0.6.3-py2.6.egg/bashplotlib/histogram.py", line 125
    hist = {i: 0 for i in range(len(bins))}
                   ^
SyntaxError: invalid syntax

**Comment by Sutarmin** [comment]
It's a dict comprehension is used here (appeared in Python 2.7). For Python 2.6 it works so:

    hist = dict((i, 0) for i in range(len(bins)))

You can change it localy or use later version of Python.

**Comment by glamp** [comment]
fixed in 40b56cdbb26d111494af71428c4cee3a22c66a68. i'll do a point release as well.

**Comment by arm2arm** [comment]
nice! thank you for fixing!
